packages/cli/src/modules/insights/database/repositories/insights-by-period.repository.ts

```diff
@@ -225,15 +225,15 @@ export class InsightsByPeriodRepository extends Repository<InsightsByPeriod> {
 				: dbType === 'postgresdb'
 					? sql`
 								SELECT
-								(CURRENT_DATE - INTERVAL '7 days')::timestamptz AS current_start,
-								CURRENT_DATE::timestamptz AS current_end,
-								(CURRENT_DATE - INTERVAL '14 days')::timestamptz AS previous_start
+								(NOW() - INTERVAL '7 days')::timestamptz AS current_start,
```

Comment:
Is it natural to expect the start to be based on the current time of the day? 🤔 Now the first day will be partial, which feels strange to me if we're fetching this data in `day` sized bucketsdays

Reply:
@Cadiac thanks for the review. I see your point. but here the goal is to fetch the comparison between the last 7 days sliding period and the one before. I think users expect the current day data to appear on that (especially for new users that launch workflows and want to see insights right away, it would feel weird IMO if they have to wait for the next day before seeing stats on their overview panel)
![image](https://github.com/user-attachments/assets/318e5443-af49-4501-90d5-cf64bf0d089d)

Reply:
Right, so this is how its shown - in that case the implementation seems correct to me. Looks like I misread the query at `insights-by-period.repository.ts` and thought there was grouping by days, but there isn't. LGTM!